README.md

```diff
@@ -1,3 +1,73 @@
 # Remote environment client
 
-This 
+A command line tool to help with using Continuous Pipe as a remote development environment.
+
+This helps to set up Kubectl, create, build and destroy remote environments and keep files
+in sync with the local filesystem.
+
+## Installation
+
```

Comment:
The ideal to me would be that we create [GitHub releases](https://github.com/continuouspipe/remote-environment-client/releases) with an artifact being the bash script containing the Git tag as a comment in order to be able to see which version do we have. (or even a command would be fab tbh).

But yeah, simply `curl`ing ,`mv`ing and `chmod`ing the bash script sounds good to me, nah?